test_requirements.txt

```diff
@@ -8,3 +8,4 @@ git+https://github.com/PulpQE/pulp-smash.git#egg=pulp-smash
 # pin pydocstyle until https://gitlab.com/pycqa/flake8-docstrings/issues/36 is resolved
 pydocstyle<4
 pytest
+mazer
```

Comment:
Is this needed? I feel like in a test environment, we'd install the packages from setup.py and mazer is there already:

https://github.com/pulp/pulp_ansible/blob/master/setup.py#L5

Reply:
You are right. I removed the other one.